dev/io.openliberty.microprofile.reactive.streams.operators30.internal_fat_tck/build.gradle

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+configurations {
+  jcommander
+}
+	
+dependencies {
+  jcommander 'com.beust:jcommander:1.72'
+}
+	
+task addjcommander(type: Copy) {
+  from configurations.jcommander
+  into new File(autoFvtDir, 'publish/servers/ReactiveStreams30TCKServer/lib/global')
+}
+
+addRequiredLibraries {
+  dependsOn addjcommander
+}
+
```

Comment:
Not sure if this is needed. It's a dependency of TestNG that we're for some reason pinning to a specific version.